### PR TITLE
Update development.md

### PR DESCRIPTION
**Pull request by xp44mm**
os:windows

```bash
error: option '--open <value>' argument missing
```

_describe your changes..._

- [ ] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [ ] Make sure your PR complies with the [writer's guide][2].
- [ ] Review the diff carefully as sometimes this can reveal issues.
- [ ] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/content/guides/development.md [hunk 1]
```diff
@@ -274,7 +274,7 @@ __package.json__
     "scripts": {
       "test": "echo \"Error: no test specified\" && exit 1",
       "watch": "webpack --watch",
-+     "start": "webpack serve --open",
++     "start": "webpack serve",
```
Comment (chenxsan) [inline src/content/guides/development.md:277]: cc @evilebottnawi What do you think here? Shall we remove`--open` temporarily? It won't work until webpack-dev-server v4 is out according to https://github.com/webpack/webpack-cli/issues/2001#issuecomment-720236731

File: src/content/guides/development.md [hunk 2]
```diff
@@ -295,7 +295,7 @@ __package.json__
   }
 ```
 
-Now we can run `npm start` from the command line and we will see our browser automatically loading up our page. If you now change any of the source files and save them, the web server will automatically reload after the code has been compiled. Give it a try!
+Now we can run `npm start --open` from the command line and we will see our browser automatically loading up our page. If you now change any of the source files and save them, the web server will automatically reload after the code has been compiled. Give it a try!
```
Comment (chenxsan) [inline src/content/guides/development.md:298]: Please revert this as it doesn't fix the problem.

File: src/content/guides/development.md (hunk 1 above)
Comment (evilebottnawi) [reply src/content/guides/development.md:277]: Keep `--open`, I will fix it in near future (2-3 days)

**Comment by chenxsan** [comment]
Closing this as we would prefer keeping `--open`.

**Pull request closed without merging**